test/kernel-headers/source/runners/3-fetch.sh

```diff
@@ -8,4 +8,12 @@ set -x
 
 vagrant ssh -- -- ./uninstall-kernel-headers.sh
 vagrant ssh -- -- sudo rm -rf /var/cache/ebpf_net/kernel-headers || true
-vagrant ssh -- -R "5000:localhost:5000" -- ./agent.sh
+
+if [ $# -eq 2 ]
```

Comment:
Are all of these ifs of # args necessary - can't you pass all args with "$@" ?

Reply:
removed.